seatunnel-core/seatunnel-core-starter/src/main/java/org/apache/seatunnel/core/starter/SeaTunnel.java

```diff
@@ -40,9 +40,11 @@ public static <T extends CommandArgs> void run(Command<T> command) throws Comman
             command.execute();
         } catch (ConfigRuntimeException e) {
             showConfigError(e);
+            System.exit(1);
             throw e;
-        } catch (Exception e) {
+        } catch (Exception | Error e) {
             showFatalError(e);
+            System.exit(1);
```

Comment:
we should print the error statck before exit. Should we should update `showConfigError` and `showFatalError`